content/livesync/quickstart.textile

```diff
@@ -1,7 +1,244 @@
 ---
 title: Quickstart
-meta_description: "."
+meta_description: "A quickstart guide to learn the basics of integrating the Ably LiveSync product into your application."
 product: livesync
 languages:
   - javascript
 ---
+
+This quickstart will provide you with a set of instructions to get an example running locally on your machine. Following this, you'll cover each component of LiveSync found within the code of the example.
+
+h2(#example). The example
+
+In this section you will cover all the steps required to get an existing example running locally on your machine.
+
+h3(#step-0). Clone the repository
+
+Clone the "live comments repository":https://github.com/ably-labs/live-comments:
+
+```[sh]
+git clone git@github.com:ably-labs/live-comments.git
+```
+
+h3(#step-1). Create an Ably account
+
+"Sign up":https://ably.com/signup for a free account to get your own API key. Ensure that your API key includes the @subscribe@ and @publish@ "capabilities":/auth/capabilities.
+
+h3(#step-2). Set up a database
+
+You can have the freedom to choose the Postgres database you wish to use, but for the purposes of the quickstart, this step will take you through setting up a database with "Neon":https://neon.tech, a serverless platform for Postgres databases. Once created, copy the Postgres credentials to be used in the next step.
+
+h3(#step-3). Update environment variables
+
+Set up environment variables by copying them from the template:
+
+```[sh]
+ cp env.example env.local
+```
+
+Update your @env.local@ file with the following:
+
+```
+POSTGRES_URL="<YOUR_FULL_POSTGRES_URL>"
+POSTGRES_URL_NON_POOLING="<YOUR_FULL_POSTGRES_URL>"
+POSTGRES_USER=<POSTGRES_USER>
```

Comment:
Actially, we only need the `POSTGRES_URL` and `POSTGRES_URL_NON_POOLING`, so we can remove `POSTGRES_USER`, `POSTGRES_HOST`, `POSTGRES_PASSWORD` and `POSTGRES_DATABASE`